pyproject.toml

```diff
@@ -12,7 +12,9 @@ python = "^3.9"
 cdislogging = "*"
 attrs = "==19.3.0"
 dictionaryutils = "==3.4.10"
-gdcdictionary = {git = "https://github.com/NCI-GDC/gdcdictionary.git", rev = "release/py3"}
+# set 'develop = true' to prevent over-writing by 'gen3dictionary', similar to
```

Comment:
let's delete this file: https://github.com/uc-cdis/gen3datamodel/blob/develop/setup.py